composer.json

```
@@ -85,7 +85,7 @@
             "pre-commit": [
                 "set -e",
                 "vendor/bin/parallel-lint --exclude vendor --no-progress .",
-                "vendor/bin/phpcs --cache=var/cache/phpcs.json"
+                "vendor/bin/phpcs --no-cache"
```

I don't think there's any need to change this, actually.

```suggestion
                "vendor/bin/phpcs --cache=var/cache/phpcs.json"
```